lxd/seccomp/seccomp.go

```diff
@@ -1249,6 +1249,28 @@ func FindTGID(procFd int) (uint32, error) {
 	return 0, fmt.Errorf("Task group leader ID not found")
 }
 
+func isCapableInCtInitUserns(siov *Iovec, cap C.int) (bool, error) {
+	containerInitPID := int(siov.msg.init_pid)
+	targetPID := int(siov.req.pid)
+
+	ctInitUserNS, err := os.Readlink(fmt.Sprintf("/proc/%d/ns/user", containerInitPID))
+	if err != nil {
+		return false, fmt.Errorf("Can't get userns for ct init process: %v", err)
```

Comment:
Can we change "ct" to "container" for clearer error messages please. Also please can you use %w for wrapping errors so that the underlying error can be accessed and inspected if needed by the caller.

Reply:
Fixed!